Dependencies/CMakeLists.txt

```diff
@@ -129,7 +129,9 @@ endif()
 FetchContent_MakeAvailable_With_Message(JsRuntimeHost)
 
 set_property(TARGET JsRuntime PROPERTY UNITY_BUILD false)
-set_property(TARGET AppRuntime PROPERTY UNITY_BUILD false)
+if(TARGET AppRuntime)
+    set_property(TARGET AppRuntime PROPERTY UNITY_BUILD false)
+endif()
```

Comment:
This is fine, but I'm not sure what `BABYLON_NATIVE_BUILD_APPS` has to do with this?

Reply:
When `BABYLON_NATIVE_BUILD_APPS` is set to `Off`, the `AppRuntime` target is not added.

Reply:
Hmm, whether `AppRuntime` is available is supposed to be determined by `JSRUNTIMEHOST_CORE_APPRUNTIME`.

Reply:
Oh, you're right. I listed the wrong CMake variable in the description.

Reply:
Description fixed and commit message updated.